documentation/domains/Domain.md

```diff
@@ -136,6 +136,7 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 | `initContainers` | Array of [Container](k8s1.13.5.md#container) | Initialization containers to be included in the server Pod. See `kubectl explain pods.spec.initContainers`. |
 | `labels` | Map | The labels to be added to generated resources. The label names must not start with "weblogic.". |
 | `livenessProbe` | [Probe Tuning](#probe-tuning) | Settings for the liveness probe associated with a WebLogic Server instance. |
+| `maxReadyWaitTimeSeconds` | number | The maximum time in seconds that the operator waits for a WebLogic server pod to get into ready state before it considers the server pod failed. Defaults to 1800 seconds. |
```

Comment:
"WebLogic server" -> "WebLogic Server"
"to get into" -> "to reach the"
last "server pod" -> "pod"